boards/arm/swan_r5/board.c

```diff
@@ -9,20 +9,20 @@
 
 static int board_swan_init(const struct device *dev)
 {
-	const struct gpio_dt_spec gpio4 =
-		GPIO_DT_SPEC_GET(DT_PATH(zephyr_user), pull_up_gpios);
-	const struct gpio_dt_spec gpio6 =
-		GPIO_DT_SPEC_GET(DT_PATH(zephyr_user), no_pull_gpios);
+	const struct gpio_dt_spec gpioe6 =
+		GPIO_DT_SPEC_GET_BY_IDX(DT_PATH(zephyr_user), push_pull_gpios, 0);
+	const struct gpio_dt_spec gpioc6 =
+		GPIO_DT_SPEC_GET_BY_IDX(DT_PATH(zephyr_user), push_pull_gpios, 1);
```

Comment:
suggestion: rename such variables to not include gpio pin information, this code is generic in the end.